stixcore/idb/manager.py

```diff
@@ -23,22 +21,67 @@
 IDB_VERSION_DELIM = "."
 IDB_VERSION_HISTORY_FILE = "idbVersionHistory.json"
 
+IDB_FORCE_VERSION_KEY = '__FORCE_VERSION__'
```

Comment:
with a module level variable if there happen to be two instances of the class they would both access the same variable I know it's probably unlikely but any issues in making it a class variable?

Reply:
no it is just a constant name in order to lookup in a dictionary. each instance has it own dict.